tests/units/anta_tests/routing/test_generic.py

```diff
@@ -304,6 +304,105 @@
+                {"prefix": "10.100.0.128/31", "vrf": "MGMT", "strict": True, "nexthops": ["10.100.0.8", "10.100.0.10"]},
+            ]
+        },
+        "expected": {"result": "success"},
+    },
+    {
+        "name": "failure-not-configured",
+        "test": VerifyRouteEntry,
+        "eos_data": [
+            {"vrfs": {"default": {"routes": {}}}},
+            {"vrfs": {"MGMT": {"routes": {}}}},
+        ],
+        "inputs": {
+            "route_entries": [
+                {"prefix": "10.10.0.1/32", "vrf": "default", "strict": True, "nexthops": ["10.100.0.8", "10.100.0.10"]},
+                {"prefix": "10.100.0.128/31", "vrf": "MGMT", "strict": True, "nexthops": ["10.100.0.8", "10.100.0.10"]},
+            ]
+        },
+        "expected": {
+            "result": "failure",
+            "messages": [
+                "Following route entry(s) or nexthop path(s) not found or not correct:\n"
+                "{'10.10.0.1/32': {'default': 'Not configured'}, '10.100.0.128/31': {'MGMT': 'Not configured'}}"
+            ],
+        },
+    },
+    {
+        "name": "failure-strict-failed",
+        "test": VerifyRouteEntry,
+        "eos_data": [
+            {
+                "vrfs": {
+                    "default": {
+                        "routes": {
+                            "10.10.0.1/32": {
+                                "vias": [{"nexthopAddr": "10.100.0.8", "interface": "Ethernet1"}, {"nexthopAddr": "10.100.0.10", "interface": "Ethernet2"}],
+                            }
+                        }
+                    }
+                }
+            },
+            {
+                "vrfs": {
+                    "MGMT": {
+                        "routes": {
+                            "10.100.0.128/31": {
+                                "vias": [{"nexthopAddr": "10.100.0.8", "interface": "Ethernet1"}, {"nexthopAddr": "10.100.0.10", "interface": "Ethernet2"}],
+                            }
+                        }
+                    }
+                }
+            },
+        ],
+        "inputs": {
+            "route_entries": [
+                {"prefix": "10.10.0.1/32", "vrf": "default", "strict": True, "nexthops": ["10.100.0.8", "10.100.0.10", "10.100.0.11"]},
+                {"prefix": "10.100.0.128/31", "vrf": "MGMT", "strict": True, "nexthops": ["10.100.0.8", "10.100.0.10", "10.100.0.11"]},
+            ]
+        },
+        "expected": {
+            "result": "failure",
+            "messages": [
+                "Following route entry(s) or nexthop path(s) not found or not correct:\n"
+                "{'10.10.0.1/32': {'default': 'Expected only `10.100.0.8, 10.100.0.10, 10.100.0.11` nexthops should be listed but "
+                "found `10.100.0.8, 10.100.0.10` instead.'}, '10.100.0.128/31': {'MGMT': 'Expected only `10.100.0.8, 10.100.0.10, "
+                "10.100.0.11` nexthops should be listed but found `10.100.0.8, 10.100.0.10` instead.'}}"
+            ],
```

Comment:
add the unit test where strict is true and length of actual and expected nexthop is also same but the addresses are not same.